src/Network/Freddy/Result.hs

```diff
@@ -0,0 +1,8 @@
+module Network.Freddy.Result (Result (..)) where
+
+data Result = Success | Error
+
+instance Show Result where
```

Comment:
these are used in ruby/node freddy to display errors. e.g. https://github.com/salemove/freddy/blob/82f85b97bbd1c2abc3bdfb09acb2d7306a41fc31/lib/freddy/sync_response_container.rb#L32 . Result here is used on line 37 https://github.com/salemove/freddy-hs/pull/1/files#diff-b859a46c2e03a2d192cd5ebbb5eeb447R37